### DISCUSSION TITLE
Support for css writing-modes

### DISCUSSION THREAD

**Discussion by skylerknight** [tailwindlabs/tailwindcss#9240] [Ideas]
It would be nice if Tailwind also supported css writing-modes:

```css
/* Keyword values */
writing-mode: horizontal-tb;
writing-mode: vertical-rl;
writing-mode: vertical-lr;

/* Global values */
writing-mode: inherit;
writing-mode: initial;
writing-mode: revert;
writing-mode: revert-layer;
writing-mode: unset;
```

I can't seem to find utility classes in the docs regarding `writing-mode` or `text-orientation`.
Something like this might work:

```html
<p class="writing-vrl orientation-upright">안녕하세요</p>
```

**Comment by mrienstra** [comment]
Related discussions:
https://github.com/tailwindlabs/tailwindcss/discussions/2494
https://github.com/tailwindlabs/tailwindcss/discussions/5237
https://github.com/tailwindlabs/tailwindcss/discussions/6845
